app/src/main/java/com/example/awesomelanguagelearning/chooseLanguage/ui/views/ChooseLanguageOverviewPage.kt

```diff
@@ -15,21 +15,21 @@ import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
 import com.example.awesomelanguagelearning.R
 import com.example.awesomelanguagelearning.chooseLanguage.domain.entity.ChooseLanguagePageState
+import com.example.awesomelanguagelearning.chooseLanguage.ui.ChooseLanguageEvent
 import com.example.awesomelanguagelearning.core.ui.theme.AppTheme
 import com.example.awesomelanguagelearning.core.ui.views.HorizontalSpacer
 import com.example.awesomelanguagelearning.core.ui.views.TextButton
 import com.example.awesomelanguagelearning.core.ui.views.TextTitle
 
 @Composable
-fun ChooseLanguageLastPage(
+fun ChooseLanguageOverviewPage(
     state: ChooseLanguagePageState,
-    onNextClick: () -> Unit = {}
+    onEvent: (ChooseLanguageEvent) -> Unit = {}
 ) {
```

Comment:
@igel911 same here, let's add `modifier` here

Reply:
@slothdeboss Is it a phobia or something?